gke/variables.tf

```diff
@@ -105,14 +105,14 @@ variable "enable_dashboard" {
 
 variable "private_k8s_endpoint" {
   type        = bool
-  default     = true
-  description = "By default, the Kubernetes endpoint is only accessible via the bastion host. Set to false if you want access via the public internet. You can use IP whitelisting using `allowed_cidr_blocks` to tighten access for both cases."
+  default     = false
+  description = "Setting this to true will disable access to the k8s API via the public internet. You will need a bastion or VPN to operate the k8s cluster"
```

Comment:
This is fairly different than EKS (only one variable, only for private and default false), is it worth making the options similar on each platform? Or will that introduce breaking changes?

Reply:
That's just something that's not supported by Terraform for GKE. Either enable private endpoint and disable public endpoint or use public endpoint.

Reply:
Ah! I see! Thanks